### PR TITLE
Task-3

### PR DESCRIPTION
**Pull request by Feralwater**

### CODE REVIEW

File: src/actions/appointments.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
+export function getAppointments() {
```
Comment (mkukharchuk) [inline src/actions/appointments.ts]: Please rename this folder from 'action' to smth else (like mock-data, or mock-helper)

File: src/actions/doctors.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+];
+
+export function getMe() {
+    // return undefined
+    return users[1]
+}
+
+// export function getdoctors() {
+//     // return undefined
+//     return users
```
Comment (mkukharchuk) [inline src/actions/doctors.ts]: Please get rid of commented code

File: src/components/button/Button.tsx [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+}
+
+const Button: React.FC<ButtonPropsType> = ({
+                                               children,
+                                               type,
+                                               disabled,
+                                               onClick,
+                                               styledComponent
+                                           }) => {
+    const StyledButton = styled(styledComponent)``;
```
Comment (mkukharchuk) [inline src/components/button/Button.tsx]: For what is this wrapper?

File: src/components/calendar/calendar.ts [hunk 4]
```diff
@@ -0,0 +1,75 @@
+const DAYS_IN_WEEK = 7;
```
Comment (mkukharchuk) [inline src/components/calendar/calendar.ts]: Move helper functions to utils and constants to constants

File: src/components/forms/RestorePasswordForm.tsx [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+type Values = {
+    email: string
+}
+
+const RestorePasswordForm = () => {
+    const validationSchema = Yup.object({
+        email: Yup.string()
+            .email('Email is invalid')
+            .required('Email is required'),
+    })
```
Comment (mkukharchuk) [inline src/components/forms/RestorePasswordForm.tsx]: Move this scheme to separate file please

File: src/components/message/MessageTemplate.tsx [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+                    ? <ErrorMessageTitleText>Success message goes here</ErrorMessageTitleText>
+                    : <ErrorMessageTitleText>Error message goes here</ErrorMessageTitleText>
+                }
+                <Button type={"button"} onClick={() => setShowErrorMessage(false)}
+                        styledComponent={CloseButton}><Close/></Button>
+            </ErrorMessageTitle>
+            <ErrorMessageText>
+                {isSuccess
+                    ? "We show this message if something awesome has happened. You are awesome too"
+                    : "We show this message if something irreparable has happened. But there is nothing irreparable"
```
Comment (mkukharchuk) [inline src/components/message/MessageTemplate.tsx]: All texts move to dictionaty.js(ts)

File: src/components/button/Button.tsx (hunk 3 above)
Comment (Feralwater) [reply src/components/button/Button.tsx]: It is needed to send styled component as a prop. I`ll delete it in task-4

**Pull request closed without merging**